tests/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages.Tests/MSBuildStepTestCollection.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.UpgradeAssistant.Fixtures;
+using Xunit;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Packages.Tests
+{
+    /// <summary>
+    /// Class (which is never instantiated) for collecting package step tests that
+    /// need test fixtures to register MSBuild and make NuGet packages available.
+    /// </summary>
+    [CollectionDefinition(Name)]
+    public class MSBuildStepTestCollection : ICollectionFixture<MSBuildRegistrationFixture>
```

Comment:
Why did you need to add this?

Reply:
Added because the `IsApplicable()` logic checks for a [transitive dependency](https://github.com/dotnet/upgrade-assistant/blob/65a12f4e5efa98d052efe0c7ab533e117bc9e3d0/src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/Analyzers/TransitiveDependencyExtensions.cs#L15) which ultimately calls `NuGetFramework.Parse(tfm.Name);`

Reply:
Ah. Hold on for my open PR to merge then it won't

Reply:
It's merged now, so you shouldn't need any of the msbuild related stuff you added